impl/src/settings.rs

```diff
@@ -63,6 +63,12 @@ pub struct RazeSettings {
   #[serde(default)]
   pub targets: Option<Vec<String>>,
 
+  /**
+   * A list of binary dependencies
+   */
+  #[serde(default)]
+  pub binaries: HashMap<String, cargo_toml::Dependency>,
```

Comment:
Optional Nit: Not sure that this is self-evident in at usage site inside toml. Maybe would prefer `binary_deps` instead?

I don't feel strongly about this suggestion.

Reply:
Updated